packages/core/src/util/TaskScheduler/TaskScheduler.ts

```diff
@@ -28,7 +28,7 @@ const logger = logdown('@wireapp/core/TaskScheduler', {
 
 type ScheduleTaskParams = {
   task: () => void;
-  firingDate: number;
+  firingDate: number | string;
```

Comment:
What type of string do we expect here? number as a string?

Reply:
DateTime - String

Reply:
Since we're converting it to number anyway, can't we just accept number and force consumer to convert it?